src/Simplex/Messaging/Agent.hs

```diff
@@ -250,17 +251,22 @@ processSMPTransmission c@AgentClient {sndQ} (srv, rId, cmd) = do
             A_MSG body -> do
               logServer "<--" c srv rId "MSG <MSG>"
               -- TODO check message status
-              withStore $ \st -> createRcvMsg st connAlias body senderMsgId senderTimestamp srvMsgId srvTs
-              -- TODO either:
-              -- TODO   - pass MSG{.., msg = generatedRcvMsg} to notify,
-              -- TODO   - retrieve internalTs from generatedRcvMsg,
-              -- TODO   - pass recipientTs to createRcvMsg (second cleanest),
-              -- TODO   - remove m_recipient from MSG (might be the cleanest?).
               recipientTs <- liftIO getCurrentTime
+              recipientId <- withStore $ \st ->
+                createRcvMsg
+                  st
+                  connAlias
+                  body
+                  recipientTs
+                  senderMsgId
+                  senderTimestamp
+                  srvMsgId
+                  srvTs
```

Comment:
a single long line can be easier to read

Reply:
agreed.. it just goes beyond 120 length which becomes a bit too much.. for now w/e I guess, we have some signatures in Store that also exceed 120. Will return to it once we set up line length rules